examples/vector_return/src/main.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python2
+import sys, ctypes
+from ctypes import pointer, POINTER, c_size_t
+
+prefix = {'win32': ''}.get(sys.platform, 'lib')
+extension = {'darwin': '.dylib', 'win32': '.dll'}.get(sys.platform, '.so')
+lib = ctypes.cdll.LoadLibrary(prefix + "vector_return" + extension)
+
+lib.counter_generate.argtypes = (c_size_t, c_size_t, POINTER(POINTER(c_size_t)))
+lib.counter_generate.restype = ctypes.c_size_t
+lib.counter_free.argtypes = ((POINTER(c_size_t)), c_size_t)
```

Comment:
These are tuples, so I'm being explicit about the declaration. I haven't see much python where the parens around a tuple are omitted, so I didn't even think to omit them here.

If you're against this style, I can omit them.